[nrfconnect] Add support for nRF52840 Dongle to lighting-app (#77)
Damian-Nordic/connectedhomeip: examples/lighting-app/nrfconnect/README.md

> 

Reviewer: Remove "(called rendezvous)" from line 80?

Author: Can we file an issue for such changes and make it across all our docs? It's unrelated to this change, so I would prefer to avoid side changes.

Reviewer: You can skip these changes for now.

| Hardware platform                                                                                 | Build target               | Platform image                                                                                                                                          |
| ------------------------------------------------------------------------------------------------- | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)         | `nrf52840dk_nrf52840`      | <details><summary>nRF52840 DK</summary><img src="../../platform/nrfconnect/doc/images/nRF52840_DK_info-medium.jpg" alt="nRF52840 DK"/></details>        |
| [nRF5340 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF5340-DK)           | `nrf5340dk_nrf5340_cpuapp` | <details><summary>nRF5340 DK</summary><img src="../../platform/nrfconnect/doc/images/nRF5340_DK_info-medium.jpg" alt="nRF5340 DK"/></details>           |

> | [nRF52840 Dongle](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-Dongle) | `nrf52840dongle_nrf52840`  | <details><summary>nRF52840 Dongle</summary><img src="../../platform/nrfconnect/doc/images/nRF52840_Dongle-medium.jpg" alt="nRF52840 Dongle"/></details> |

Reviewer: Is the information in the `Device UI` section compatible with the nRF52840 Dongle? I guess at least the PCB names might differ... And it would be good to add some information about PCB elements only available on the DKs, like the J-Link USB, no?

monitor the state of the device. These correspond to PCB components on the
platform image.

> > **Important**:
> >
> > Please note that the following elements are missing on the nRF52840 Dongle:
> > **Button 2**, **Button 3**, **Button 4**, **SEGGER J-Link USB port** and
> > **NFC port with antenna attached**. Logs from a nRF52840 Dongle can be
> > collected using **nRF USB port** instead of **SEGGER J-Link USB port**.
> > Functionalities associated with the remaining missing elements are
> > inaccessible.

Reviewer: ```suggestion
> **Note**:
>
> The following Device UI elements are missing on the nRF52840 Dongle:
> **Button 2**, **Button 3**, **Button 4**, **SEGGER J-Link USB port**, and
> **NFC port with antenna attached**. You can collect logs from th nRF52840 Dongle
> using the **nRF USB port** instead of the **SEGGER J-Link USB port**.
> Functionalities associated with the remaining missing elements are
> inaccessible.
```

> ## Flashing and debugging

Reviewer: Please change `flashing` to `programming` to be consistent with the Nordic terminology. This is a global change for the whole file.

        $ west debug

<a name="nrf52840dongle_flashing"></a>

### Flashing on the nRF52840 Dongle

Visit

> [Programming and Flashing nRF52840 Dongle](https://docs.zephyrproject.org/latest/boards/arm/nrf52840dongle_nrf52840/doc/index.html#programming-and-debugging)

Reviewer: The page name is different: `Programming and debugging`
